### PR TITLE
added brotli compression support

### PR DESCRIPTION
**Pull request by pjordaan**

### CODE REVIEW

File: src/EventListener/BrotliListener.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+     */
+    public function onPostWrite(FileEvent $event): void
+    {
+        $file            = $event->getFile();
+        // if the file is already compressed with brotli/gzip, do not compress it again as we do not serve files
+        // like .br.gz.br
+        if (preg_match('/\.(gz|br)$/', $file->path)) {
+            return;
+        }
+        $content         =$event->getContent();
```
Comment (nicoschoenmaker) [inline src/EventListener/BrotliListener.php]: Space after `=`

**Comment by nicoschoenmaker** [comment]
Approved after build works ofc, brotli compression test appears to be failing.

File: package.json [hunk 2]
```diff
@@ -12,6 +12,7 @@
     "url": "https://github.com/hostnet/resolver-lib/issues"
   },
   "dependencies": {
+    "brotli": "1.3.2",
```
Comment (yannickl88) [inline package.json]: We might consider moving this somewhere else in the future, not sure if NodeJS has a `suggest` equivalent. Bit of a shame to have these dependencies if you are not using them.

I made #63 to have a reminder.

Comment (pjordaan) [reply package.json]: well, the first start was moving all the require statements in the processor.js, which this PR already did.....

**Pull request merged**